### DISCUSSION TITLE
How do how switch domains on Azuracast?

### DISCUSSION THREAD

**Discussion by pwrj** [AzuraCast/AzuraCast#4581] [Q&A]
How do how switch domains on Azuracast? I want to replace domain1.com with domain2.com

**Comment by BusterNeece** [comment]
@pwrj Re-running `./docker.sh setup-letsencrypt` should help you with getting the new certificate going. If you have to update the base URL, it may also be helpful to run something like:
```
./docker.sh cli azuracast:settings:set preferBrowserUrl true
```

**Reply by pwrj** [reply]
@SlvrEagle23 Thank you so much!
